.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @digorithm @arboleya @Torres-ssf @Dhaiwat10 @danielbate @nedsalk @petertonysmith94
+* @digorithm @arboleya @Torres-ssf @Dhaiwat10 @danielbate @nedsalk @petertonysmith94 @maschad
```

Comment:
@arboleya this is invalid until chad has org access

Reply:
I have access now so this should be fine.